### DISCUSSION TITLE
需要最新安卓安装包

### DISCUSSION THREAD

**Discussion by EdwardChoe** [getlantern/lantern-forum-cn#214] [General]
如题，我是三星note10+手机，找到的各种apk安装包都无法正确安装。需要一个可用的安装包。
![Weixin Image_20240304162728](https://github.com/getlantern/lantern-forum-cn/assets/20028119/fa37582a-38ab-43ab-968b-e3817362e649)

**Comment by MaxMKMax** [comment]
@EdwardChoe 您好。您可以尝试打开此常见问题解答链接来查看如何下载和安装该应用程序吗？我认为此链接可能有助于解决您面临的问题。
https://lantern.io/zh-Hans/faq

**Reply by EdwardChoe** [reply]
已经尝试过，在安卓系统更新之后，该apk文件已经无法正常安装。

**Reply by MaxMKMax** [reply]
@EdwardChoe 您好，好的. 明白的! 谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。
